docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+
+## Getting started
+
+To demonstrate how to get up and running with TLS in Kubernetes, we’ll use a sample `yaml` configuration file for a one-node cluster.
+
+To enable TLS we need to add a listener entry under the API fields in our yaml configuration file, and then specify the subfield `tls` where we can enable TLS.
+
+For example, check that `kafkaApi` now have a new flag that says `tls: enabled: true`.
+
+Save this file as `yaml`:
+
+```yaml
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Cluster
+metadata:
+ name: cluster-sample-tls
+spec:
+ image: "vectorized/redpanda"
+ version: "latest"
+ replicas: 1
+ resources:
+   requests:
+     cpu: 1
+     memory: 1.2G
+   limits:
+     cpu: 1
+     memory: 1.2G
+ configuration:
+   rpcServer:
+     port: 33145
+   kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+   pandaproxyApi:
+    - port: 8082
+      tls:
+        enabled: true
+   adminApi:
+   - port: 9644
+   developerMode: true
+```
+
+We’ll use the Kafka API to go through the TLS configuration, but you can use similar settings to set up TLS for the other APIs. For this guide, we assume external connectivity is disabled unless otherwise stated.
+
+The operator supports a single listener with TLS per API. Additionally, if you have two listeners for the same API and one of them has external connectivity enabled, the operator assumes that the latter listener has TLS enabled.
+
+Providing an issuer or certificate to the operator is only available for the Kafka API. 
+
+### Client Auth (mTLS)
+
+When you enable TLS, the client has the option of verifying the identity of the server, but to do this the client must have the CA’s certificate available in a trusted store.
+
+We can then enable client authentication (mTLS) on the same listener by setting `requireClientAuth` to `true`.
+
+So for example, we can edit our previous `yaml` file to add this new setting: 
+
+```yaml
+kafkaApi:
+    - port: 9092
+    tls:
+        enabled: true
+        requireClientAuth: true
+```
+
+When you enable client authentication, you must specify a key-pair (certificate and key). As we will later see, the operator can also generate the key-pair.
+
+When you finish setting up your configuration, apply it by running:
+
+```bash
+kubectl apply \
+-f <your-file-path.yaml>
+```
+
+Remember to change to the actual file path that you used. 
+
+
+## Managing Certificates
+
+### Certificates Creation
+
+The Redpanda operator uses [cert-manager](https://cert-manager.io) to generate certificates. 
+
+By default, when you enable TLS, the Redpanda operator generates a root certificate for each API and uses that to generate leaf certificates for the nodes and client. You can check what certificates were generated in the section down below. 
+
+Specifically for the Kafka API, the operator also offers two options: 
+
+1) provide an existing certificate issuer; 
+   
+2) provide a node certificate. 
+
+This functionality is only available for the Kafka API.
```

Comment:
I don't think you need this since you mentioned above that this is for the Kafka api. Do we need to say anything about how this works in the other apis? It's always the redpanda-generated certificate?

Reply:
We are stressing this because it only works for the Kafka API and not the others (Admin, PandaProxy). 
It's a good idea to ask for feedback on this @vsaraswat

Reply:
We did mention in line 66 that this is only available for Kafka API. I think it is worth emphasizing which other APIs are not available (e.g. Panda Proxy and Schema Registry) because we get these questions every so often. Not sure it needs to appear in two different locations in the doc.

Reply:
```suggestion
This functionality is not available for the PandaProxy, and Admin API. If you want to see this feature, feel free to [open an issue for us](https://github.com/vectorizedio/redpanda/issues).
```